seatunnel-connectors-v2/connector-paimon/src/test/java/org/apache/seatunnel/connectors/seatunnel/paimon/utils/RowConverterTest.java

```diff
@@ -248,6 +261,34 @@ public void seaTunnelToPaimon() {
         InternalRow reconvert =
                 RowConverter.reconvert(seaTunnelRow, seaTunnelRowType, getTableSchema(30, 8));
         Assertions.assertEquals(reconvert, internalRow);
+
+        isCaseSensitive = true;
```

Comment:
Does it worked?

Reply:
Yes, it works, otherwise there would be no uppercase field generated. I iterate through the fields array to make every field generate an uppercase field and check.

Reply:
But I found it only used on https://github.com/apache/seatunnel/pull/7549/files/9b67afc0c506b2f64dfad3d1bcc25e23e8d02b31#diff-169060bad0a00b78c4decabbd35d6b27999c1608d8e7aba4ef707622f06ac8c9R142
It be triggered before test case started. So I wonder why it works.

Reply:
oh, I got it. You trigger `before` method after change `isCaseSensitive `.